docs/guides/user/sigma.md

```diff
@@ -23,34 +23,33 @@ This will show a table with all Sigma rules installed on a system. You can searc
 
 - Rule ID
 - Title
-- Elastic Search query (ES Query)
+- OpenSearch Search query (ES Query)
```

Comment:
ES Query -- OS Query

Reply:
ah nevermind we need to refactor it in the module then as well.

Reply:
This is still Elasticsearch due to how Sigma still uses that.